docs/ref/databases.txt

```diff
@@ -916,11 +916,16 @@ To enable the JSON1 extension you can follow the instruction on
 Oracle notes
 ============
 
-Django supports `Oracle Database Server`_ versions 19c and higher. Version 7.0
-or higher of the `cx_Oracle`_ Python driver is required.
+Django supports `Oracle Database Server`_ versions 19c and higher. Version 1.1.0
```

Comment:
`1.2.0`?
```suggestion
Django supports `Oracle Database Server`_ versions 19c and higher. Version 1.2.0
```

Reply:
@felixxm Version 1.1.0 is ok. Version 1.1.0 or higher is required.

Reply:
So this patch is consistent, we have different versions listed in different places.

Reply:
I did not understand what do you mean by different versions listed in different places.

Reply:
> I did not understand what do you mean by different versions listed in different places.

Please check this patch:
- `docs/releases/5.0.txt` -> version 1.2.0
- `tests/requirements/oracle.txt`  -> version 1.2.0
- `docs/ref/databases.txt` -> version  1.1.0

Reply:
Thanks, understood now. Let us stick to 1.2.0 as DB_TYPE_OBJECT is supported from this version.